driver/razerkbd_driver.c

```diff
@@ -434,8 +435,17 @@ static ssize_t razer_attr_write_mode_game(struct device *dev, struct device_attr
 {
     struct usb_interface *intf = to_usb_interface(dev->parent);
     struct usb_device *usb_dev = interface_to_usbdev(intf);
+    struct razer_report report = {0};
     unsigned char enabled = (unsigned char)simple_strtoul(buf, NULL, 10);
-    struct razer_report report = razer_chroma_standard_set_led_state(VARSTORE, GAME_LED, enabled);
+
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_HUNTSMAN_V2:
+        report = razer_chroma_standard_set_led_state(NOSTORE, GAME_LED, enabled);
```

Comment:
The packet capture indicated Synapse was not using VARSTORE. I could just modify the transaction ID and continue using VARSTORE here if you think that best.